.project

```diff
@@ -38,4 +38,15 @@
 		<nature>net.sourceforge.pmd.eclipse.plugin.pmdNature</nature>
 		<nature>edu.umd.cs.findbugs.plugin.eclipse.findbugsNature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
Please revert unrelated changes. 

(I need to remove them from the project in a separate PR)